pandas/tests/frame/test_arithmetic.py

```diff
@@ -138,3 +143,252 @@ def test_ops_frame_period(self):
                             'B': np.array([16, 16], dtype=object)})
         tm.assert_frame_equal(df2 - df, exp)
         tm.assert_frame_equal(df - df2, -exp)
+
+
+class TestFrameFlexArithmetic(object):
+    """Tests for DataFrame flex operations, mostly focused on alignment
+    and broadcasting.
+    """
+    def test_df_flex_add_int(self):
+        frame = pd.DataFrame(tm.getSeriesData())
```

Comment:
I wonder if we should make `pd.DataFrame(tm.getSeriesData())` its own `tm` method given how often you use it.